.gitignore

```diff
@@ -145,3 +145,6 @@ pyvenv.cfg
 
 # Certificate files
 *.pem
+
+# AI
+AGENTS.md
```

Comment:
Let me tackle this in a separate PR. I want to set this up for a agent coding tools